### PR TITLE
Rated page

### LINKED ISSUE #75: create rated cocktails page
RatedCocktails

### PR DESCRIPTION
**Pull request by zachjamesgreen**
Added rated page that calls to the backend at route /api/v1/cocktails/rated. I have some code already written up for the backend just need to throw in a test and figure out where it will really live. ill push up the back end code here soon.

## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Build is passing
## Pull request type
Please check the type of change your PR introduces:
- [x] Feature
Issue Number: 
closes #75 
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by redferret — APPROVED** [review]

**Comment by redferret** [comment]
I'm really excited about this one.

File: src/components/rated.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  componentWillUnmount() {
+    cocktailStore.removeListener(GET_RATED_COCKTAILS, this.loadCocktails);
+  }
+
+  render() {
+    if (!this.state.isLoaded) {
+      return <div className="font-playfair font-normal text-3xl text-center mx-56">Loading...</div>;
+    } else if(this.state.cocktails.length === 0 && this.state.isLoaded){
+      return <div className="font-playfair font-normal text-3xl text-center mx-56">No rated cocktails</div>;
```
Comment (mkrumholz) [inline src/components/rated.js:46]: If they have no rated cocktails, should we maybe provide a link back to the onboard page here?

**Review by mkrumholz — COMMENTED** [review]
Looks great! I think we should link back to the onboard page if they haven't rated anything maybe, so that we can encourage folks to get that done (?)

File: src/components/rated.js (hunk 1 above)
Comment (zachjamesgreen) [reply src/components/rated.js]: added link to onboard page

**Review by mkrumholz — APPROVED** [review]
I think this looks OK! I'll probably reach out to ask about the cancelToken piece and why that's removed, but it can remain commented out for now (in the get_drinks_action) unless that will cause issues (which I'm unclear on)

**Pull request merged**